packages/ui-logic/src/weave.ts

```diff
@@ -6,12 +8,10 @@ export class Weave {
    * @param error pass in the variable from the catch block. Error and string are supported for parsing.
    */
   public static tryParseError(error: unknown): string | null {
-    const errorMessage = error instanceof Error ? error.message : typeof error === "string" ? error : "";
-
-    const normalizedErrorMessage = errorMessage.trim().replace(/^Error:\s*/, "");
+    const errorMessage = ErrorParser.normalizeErrorMessage(error);
```

Comment:
Now you do not need `Wave.tryParseError` method any more because now there is not references to this method except from `ErrorParser.tryParseError` method. So maybe you will implement all code from `ErrorParser.tryParseError`  into `Wave.tryParseError`?

Reply:
The purpose of this PR originates in [this suggestion](https://github.com/iov-one/ponferrada/pull/805#discussion_r332325342).

Reply:
Ok. But do you think we will use this method anywhere?

Reply:
Yes because I made a mistake and the Buttons component should only use Weave's. So i think [this approach](https://github.com/iov-one/ponferrada/pull/824#discussion_r334378250) would be good.